less/datepicker3.less

```diff
@@ -1,5 +1,5 @@
 .datepicker {
-	border-radius: 4px;
+	.border-radius(@border-radius-base);
```

Comment:
Hmm i'm not exactly sure why you added the 'dot' here? Or why it is in the BS2 file, should it be a "normal" property without the dot?